### ISSUE TITLE
Update Bombsquad on Snap Store for Linux

### ISSUE THREAD

**Issue by vasilislam** [efroemling/ballistica#532]
Now it is 1.6.5 on snap store, please update it to the latest
Also, how can i run or install bombsquad from the tar.gz archive in Ubuntu?

**Comment by Dliwk** [comment]
Hi! First off, this snap package is not 'official' and is maintained by someone else.

tar.gz is simply an archive with game binary and its assets; just unzip it somewhere and run `./bombsquad`.

**Comment by ixPost** [comment]
I'm on Linux!

**Comment by ixPost** [comment]
Does my BS track me?

**Comment by ixPost** [comment]
Maybe Fromleng can ask that guy to update BS? Or ask to co-own the Linux BS?

**Comment by ixPost** [comment]
On Linux the Bomb Squad Pro isn't available & there where ads but I would disconnect from the internet. Now even with the internet on, there are no ads.

**Comment by EraOSBeta** [comment]
> On Linux the Bomb Squad Pro isn't available & there where ads but I would disconnect from the internet. Now even with the internet on, there are no ads.

yeah, as far as i know bs pro can't be purchased on any platform other than android at the moment

**Comment by efroemling** [comment]
Right now purchases are limited to Android and Mac versions (and ads are limited to Android). So if you have access to either of those you can do some account linking and use those platforms to buy stuff. The plan is to launch on Steam with 2.0, and I'd hope this includes the Linux version, which should make purchases officially available there too.

I'd be happy to maintain an official Snap store version, though I don't think there's technically any way to support purchases. I just haven't had a chance to look into what it takes to set up a snap package. If anyone would like to help get that ball rolling I'd be happy to accept PRs/etc.

**Comment by brostosjoined** [comment]
https://github.com/popey/bombsquad-snap

**Comment by MatiasMGz** [comment]
Snap is terrible for thing that are not servers. Please not.
